client/src/components/product-components/Product.jsx

```diff
@@ -113,41 +113,41 @@ class Product extends React.Component {
 
   render() {
     const { currentVariation, mouseOn } = this.state;
```

Comment:
it looks like currentVariation has been commented out of state - is this causing breakage in this component as a result?

Reply:
No, it's not causing any breakage. Modified to have the backend expect same shape of response data